src/Symfony/Component/DependencyInjection/Tests/Argument/LazyClosureTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Tests\Argument;
```

Comment:
missing licence header

Reply:
Curious fabbot didn't catch this one 🤔 It's present now